src/sailing_robot/scripts/actuator_demand_rudder

```diff
@@ -22,7 +22,7 @@ def node_publisher():
     :rtype: object
```

Comment:
I find it confusing to call these scripts `actuator_`, because they're not actually talking to the hardware; they publish values which the real actuator node controls. I would reserve the name `actuator_` for the node(s) that interface with the actuators, and call these...I don't know, 'processing_'?